src/ucp/wireup/wireup.c

```diff
@@ -1655,10 +1672,15 @@ ucs_status_t ucp_wireup_init_lanes(ucp_ep_h ep, unsigned ep_init_flags,
         goto out;
     }
 
+    /* This function must be used before uct_eps are discarded. Store return
+     * value for later use. */
+    is_reconfigurable = ucp_wireup_check_is_reconfigurable(ep, &key,
```

Comment:
maybe we can move `(ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL)` inside ucp_wireup_check_is_reconfigurable, since in both places is_reconfigurable is used it is && with `(ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL)`